src/utils/general.js

```diff
@@ -8,11 +8,17 @@ let count = 0
  */
 const fixIndentation = str => {
   // Get whitespaces
-  let match = str.match(/^[ \t]*(?=\S|$)/gm)
-  // Remove first, empty item
+  const match = str.match(/^[ \t]*(?=\S|$)/gm)
+
+  // Handle oneline TTL case
+  if (match.length === 1) {
+    return {
+      text: str,
+      indentColumns: 0
+    }
+  }
+  // Remove first, empty item, and if it is a oneline TTL we already handled it
   match.splice(0, 1)
-  // Remove empty lines
-  match = match.map(x => (x.length < match[match.length - 1].length ? '  ' : x))
 
   if (!match) {
```

Comment:
What is this conditional here for? We already assumed above in `match.splice` and `match.length` that `match` exists, right? So shouldn't we either be checking this above as in, if there was no match at all, or not check it at all? The `match.splice(0, 1)` won't ever make the array `undefined` or `null` will it? Not from my experience or from a quick look at documentation anyway.